aegis_control/launch/start_drivers.launch.py

```diff
@@ -57,15 +76,31 @@ def generate_launch_description() -> LaunchDescription:
         launch_arguments=launch_args.items(),
     )
 
-    return LaunchDescription(
-        [
-            ur_driver,
-            ft_sensor_driver,
-            depthai_cameras_driver,
-        ]
-        + control_nodes
+    gripper_driver = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            PathJoinSubstitution(
+                [
+                    FindPackageShare("aegis_control"),
+                    "launch",
+                    "gripper_driver.launch.py",
+                ]
+            )
+        ),
+        launch_arguments=launch_args.items(),
     )
 
+    return [
+        ur_driver,
+        launch_after(
+            launch=control_node,
+            after=ur_comm_tool_node,
+            delay_s=0.2,  # empirically tested
+        ),
```

Comment:
Before race condition is fixed, this works with HWI reconnection:
```suggestion
        control_node
```

Reply:
resolved manually